FordLabs/PeopleMover
# 222 assignment history

## Issue
Resolves 222

## What was done
- [x] Assignment history now shows the correct start date for assignments

## How to test
1. Create multiple different assignments for a particular person, and check that the date on assignment history is correct. Recommend trying out putting person on one product vs. multiple products vs. unassigned at different points in time
